libraries/pbrlib/pbrlib_defs.mtlx

```diff
@@ -94,7 +94,7 @@
     Node: <generalized_schlick_bsdf>
     A reflection/transmission BSDF node based on a microfacet model and a generalized Schlick Fresnel curve.
   -->
-  <nodedef name="ND_generalized_schlick_bsdf" node="generalized_schlick_bsdf" nodegroup="pbr" doc="A reflection/transmission BSDF node based on a microfacet model and a generalized Schlick Fresnel curve.">
+  <nodedef name="ND_generalized_schlick_bsdf" node="generalized_schlick_bsdf" uiname="Generalized Schlick BSDF" nodegroup="pbr" doc="A reflection/transmission BSDF node based on a microfacet model and a generalized Schlick Fresnel curve.">
```

Comment:
There is an opportunity to add Hoffmann to the UI name. Same opportunities for Oren-Nayar and Sheen nodes upgraded for OpenPBR.